examples/platforms/utils/mac_frame.cpp

```diff
@@ -43,7 +43,7 @@ bool otMacFrameDoesAddrMatch(const otRadioFrame *aFrame,
     Mac::Address      dst;
     Mac::PanId        panid;
 
-    SuccessOrExit(frame.GetDstAddr(dst));
+    VerifyOrExit(frame.GetDstAddr(dst) == kErrorNone, rval = false);
```

Comment:
Does `GetDstAddr()` return anything other than `kErrorNone`?

https://github.com/openthread/openthread/blob/223935bf0c6bd02675efcb56254d21abaa1fb530/src/core/mac/mac_frame.cpp#L378

Reply:
Yes. Correct. I also noticed that `GetDstAddr()`  does not check the length and therefore tehcnilically never fails today, so this is not a (critical) bug and more of an enhancement.